<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smart_furniture</class>
 <widget class="QWidget" name="smart_furniture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/furniture/home.png);</string>
  </property>
  <widget class="QLabel" name="label_light">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/furniture/灯1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_fan">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/furniture/f.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_air">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>51</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>51</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/furniture/正常.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_T">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>71</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/furniture/温度计.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_send">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_ports"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_start">
      <property name="text">
       <string>启动服务器</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>392</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_linght">
      <property name="text">
       <string>开灯</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_fan">
      <property name="text">
       <string>开风扇</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
